README.md

```diff
@@ -46,16 +46,16 @@ export default Ember.Component.extend({
 You replace it with this:
 
 ```javascript
-import Ember from 'ember'
+import Component from '@ember/component'
 import { action, computed } from 'ember-decorators/object';
 import { service } from 'ember-decorators/service';
 
-export default class MyComponent extends Ember.Component {
```

Comment:
Do you know of any plans on making the default blueprints do that as well? Seeing that Module Unification apps will disallow nested components anyways (which would have yielded absurdly long class names), I think this is a nice feature.

In that case, I think we should use "valid" class names in our examples, like `UserProfileComponent` (to fulfill the "include a dash" constraint) instead of `MyComponent`.